src/fmu/tools/sensitivities/_designsummary.py

```diff
@@ -53,15 +53,17 @@ def summarize_design(filename, sheetname="DesignSheet01"):
 
     # Read design matrix and find realisation numbers for each sensitivity
     if filename.endswith(".xlsx"):
-        dgn = pd.read_excel(filename, sheetname)
+        dgn = pd.read_excel(filename, sheetname, engine="openpyxl")
+        dgn.dropna(axis=0, how="all", inplace=True)
+        dgn = dgn.loc[:, ~dgn.columns.str.contains("^Unnamed")]
     elif filename.endswith(".csv"):
         dgn = pd.read_csv(filename)
     else:
         raise ValueError(
-            "Design matrix filename should be on Excel or csv format"
-            " and end with .xlsx or .csv"
+            "Design matrix should be on Excel or csv format"
+            " and filename should end with .xlsx or .csv"
```

Comment:
"should -> must" is perhaps more to the point (.XLSX would even fail here, but we can live with that.)